.gitignore

```diff
@@ -7,3 +7,4 @@ root-drive.img
 TestPID.img
 build/
 testdata/logs/
+.idea
```

Comment:
I prefer to ignore IDE-specific files in a global gitignore file rather than this project-specific gitignore file.

https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
Good point! Fixed with 61d91a7